### ISSUE TITLE
"Undefined index: environment" on composer install

### ISSUE THREAD

**Issue by leberknecht** [auth0/jwt-auth-bundle#62]
Hi!

Im getting
```
In InformationHeaders.php line 128:
                                        
  Notice: Undefined index: environment
```
when i run `composer require auth0/jwt-auth-bundle:"^3.0"` on a SF4 application. Using PHP 7.2-10. I can bypass that by disabling notices, but i guess we should check if the field exists before accessing it. Or should it be `env` instead of `environment`?

![image](https://user-images.githubusercontent.com/1165177/47920313-0721ca00-deb2-11e8-9168-b9a032b58da9.png)

**Comment by joshcanhelp** [comment]
Sorry for the trouble here @leberknecht. This was an update in the PHP SDK that is having issues in these dependent libraries. Almost have a fix ready ([auth0/auth0-PHP/#304](https://github.com/auth0/auth0-PHP/pull/304)) and will release a 5.3.2 version of that in just a bit today.

**Comment by joshcanhelp** [comment]
Fixed and released in [5.3.2](https://github.com/auth0/auth0-PHP/releases/tag/5.3.2)
